book/website/community-handbook/infrastructure.md

```diff
@@ -0,0 +1,23 @@
+(ch-infrastructure)=
+# Infrastructure
+
+The Turing Way consists of many parts which need to work together to make sure the book can be build, looks correct and is available on the internet.
+There are processes to build, deploy and ensure the quality of the book.
+There is also configuration which may direct those processes or the site.
+Collectively we call these things infrastructure.
```

Comment:
```suggestion
While infrastructure can be defined in many ways, collectively we call these things infrastructure.
```
Just a suggestion for wording here - as we also use this definition to acknowledge its social elements.